src/util/types.ts

```diff
@@ -1050,13 +1050,35 @@ export interface GeometrySector {
 export type D3Scale<T> = D3ScaleContinuousNumeric<T, number>;
 
 export type AxisDomainItem = string | number | Function | 'auto' | 'dataMin' | 'dataMax';
-/** The domain of axis */
+/**
+ * The domain of axis.
+ *
+ * Can be specified in many ways:
+ * - array of numbers
+ * - with special strings like 'dataMin' and 'dataMax'
+ * - with special string math like 'dataMin - 100'
+ * - with keyword 'auto'
+ * - or a function
+ * - array of functions
+ * - or a combination of the above
+ *
+ * See https://recharts.org/en-US/api/XAxis#domain for examples
```

Comment:
How about instead adding examples here? 🤔 That would be more stable, i.e. not rely on the specific URL. Given that the website is in a different repository, we would certainly forget to change this comment if we were to change the URL / website.

Reply:
I am now working on decrypting `getDomainOfDataByKey` and when I'm done with types + tests then I should hopefully have a full set of examples that I can put here.